src/proxy/processors/push-action/pullRemote.js

```diff
@@ -1,7 +1,8 @@
-const spawnSync = require('child_process').spawnSync;
 const Step = require('../../actions').Step;
 const fs = require('fs');
 const dir = './.remote';
+const git = require('isomorphic-git');
+const http = require('isomorphic-git/http/node');
```

Comment:
```suggestion
const gitHttpClient = require('isomorphic-git/http/node');
```

I'm hesitant to just call this `http` as it's probably not useful outside of isomorphic-git itself so this name might be a bit more descriptive.
But this action is also just for cloning with this library so we could leave this as `http` too :)